nilearn/glm/regression.py

```diff
@@ -342,7 +342,7 @@ def residuals(self):
     def normalized_residuals(self):
         """Residuals, normalized to have unit length.
 
-        See :footcite:`Montgomery` and :footcite:`GVK393847152`.
+        See :footcite:`Montgomery` and :footcite:`Davidson`.
```

Comment:
Could we take this opportunity to make the citations more homogeneous, e.g. `Davidson2004` ?

Reply:
Done!